lib/pip.js

```diff
@@ -212,6 +212,8 @@ function installRequirements(targetFolder, serverless, options) {
         '-e',
         'SSH_AUTH_SOCK=/tmp/ssh_sock'
       );
+      chmod_cmd = ['chmod', '600', '/root/.ssh/id_rsa'];
```

Comment:
Is it possible to only run this command for Windows environments?
If possible, we should try to avoid modifying the permissions of user's files (since the SSH private key is mounted from the user directly).

Also, please keep in mind that there may be other, non-RSA private keys.